### PR TITLE
Fix POS refund params

### PR DESCRIPTION
**Pull request by djoksimo**
## What

- Use more logical param ordering
- Prefix POS-specific params with `Pos`

## How

- [ ] Need to update docs once the ordering is approved!

### CODE REVIEW

File: forage-android/src/main/java/com/joinforage/forage/android/pos/PosMethodParams.kt [hunk 1]
```diff
@@ -22,26 +22,26 @@ data class PosTokenizeCardParams(
...
-data class RefundPaymentParams(
+data class PosRefundPaymentParams(
     val amount: Float,
     val foragePinEditText: ForagePINEditText,
-    val metadata: Map<String, String>? = null,
     val paymentRef: String,
-    val reason: String
+    val reason: String,
+    val metadata: Map<String, String>? = null
 )
```
Comment (djoksimo) [inline forage-android/src/main/java/com/joinforage/forage/android/pos/PosMethodParams.kt:47]: @dleis612 @devinmorgan lemme know if this ordering makes sense and is conventional!

Comment (dleis612) [reply forage-android/src/main/java/com/joinforage/forage/android/pos/PosMethodParams.kt:47]: I assume alphabetical so looks good to me

**Review by dleis612 — APPROVED** [review]

File: forage-android/src/main/java/com/joinforage/forage/android/pos/PosMethodParams.kt (hunk 1 above)
Comment (devinmorgan) [reply forage-android/src/main/java/com/joinforage/forage/android/pos/PosMethodParams.kt:47]: As far as Kotlin / Android conventions, this ordering of params is perfectly. I didn't think I would have any thoughts on this but upon doing a second glance, how do you feel about the below order?
```kotlin
data class PosRefundPaymentParams(
    val foragePinEditText: ForagePINEditText,
    val paymentRef: String,
    val amount: Float,
    val reason: String,
    val metadata: Map<String, String>? = null
)
```

I'm looking at what we've been doing for [`ForageSDK`'s methods params](https://github.com/teamforage/forage-android-sdk/blob/a30ed74c6829c8aa935142825feeff1a87688c44/forage-android/src/main/java/com/joinforage/forage/android/ForageSDKInterface.kt) and it seems like there's a soft convention that:
1. the `ForageElement` is the first param
2. any `ref`s are the second param

Comment (djoksimo) [reply forage-android/src/main/java/com/joinforage/forage/android/pos/PosMethodParams.kt:47]: Will refactor to this format

Comment (devinmorgan) [reply forage-android/src/main/java/com/joinforage/forage/android/pos/PosMethodParams.kt:47]: awesome!

Comment (djoksimo) [reply forage-android/src/main/java/com/joinforage/forage/android/pos/PosMethodParams.kt:47]: 1b8a8cda2b22c231b0f7fffa063959e829526092

Comment (djoksimo) [reply forage-android/src/main/java/com/joinforage/forage/android/pos/PosMethodParams.kt]: 1b8a8cda2b22c231b0f7fffa063959e829526092

**Review by devinmorgan — APPROVED** [review]
Looks great! Thanks for these changes!!

**Pull request merged**